middlewares/sendEmail.js

```diff
@@ -0,0 +1,32 @@
+// ====================================
+// SEND WELCOME EMAIL TO NEW USERS 
+// ====================================
+
+const sendgrid = require('@sendgrid/mail');
+require('dotenv').config();
+
+//Config SendGrid
+const apiKey = process.env.SENDGRID_API_KEY;
+const ongEmail = process.env.SENDGRID_EMAIL;
+const template = process.env.SENDGRID_TEMPLATE;
+const email = process.env.TEST_EMAIL;
+
+sendgrid.setApiKey(apiKey);
+
+//Send email
+//const email = req.body.email;
+
+const message = {
+    to: email,
+    from: ongEmail,
+    templateId: template,
+};
+const sendWelcomeEmail = () => {
```

Comment:
- Elegir un nombre mas general, para que podamos usar la misma funcion para enviar distintos emails. Por ejemplo sendEmail().
- La funcion podria recibir los valores de to, from y templateId como parametros para que pueda ser reutilizada. Por ejemplo sendEmail(from, to, template) => {}
- Mas que un middleware esta funcion podria ir en helpers o en un directorio utils por ejemplo.